### PR TITLE
Fixed small typo in introduction.md

### PR DESCRIPTION
**Pull request by zakweb3**
Fixed small typo in line 65 replaced : with =

### CODE REVIEW

**Comment by BethanyG** [comment]
Hi @zakweb3 👋🏽 

Thanks for filing this PR (_and for so thoroughly reading the docs!_  😄).

You don't know this, but this `introduction.md` pulls from the [`tuples` concept doc](https://github.com/exercism/python/blob/main/concepts/tuples/about.md), so that doc likely has the same mistake.  Could you check that, and if it needs correcting, submit it as well?

Once that's sorted, I will happily merge.

Many thanks.  🚀

**Review by BethanyG — CHANGES_REQUESTED** [review]
Per my prior comment, please review the concept `about.md` to see if a change is required there as well.  Thanks!

**Review by sudhackar — APPROVED** [review]

**Comment by zakweb3** [comment]
I made the same change (replaced : with =) in line 101 in the `about.md` 
https://github.com/exercism/python/pull/3623
Thank you for considering my PR.

**Comment by BethanyG** [comment]
@sudhackar  -- please don't review PRs in this repo unless you are on the maintainer team/have reviewer permissions.  It's confusing to contributors and creates noise for maintainers.  Thanks.

@zakweb3 -- I meant for the requested change to be included in this PR, no in a separate one.  Could you please combine the two changes here and close the other PR?  _Many thanks!_

**Comment by zakweb3** [comment]
I thought you wanted to update just the 'about.md'
I was a bit confused by 
> You don't know this, but this `introduction.md` pulls from the `about.md` 

Now I did the modification on both files and deleted the other patch. 
I hope that works for you!

**Comment by BethanyG** [comment]
> I thought you wanted to update just the 'about.md'
> I was a bit confused by

    > You don't know this, but this introduction.md pulls from the about.md

Apologies for the confusing wording.  🙂 

>  Now I did the modification on both files and deleted the other patch.
>  I hope that works for you!

It does!  As soon as CI passes, I will review and merge!

File: concepts/tuples/about.md
Comment (BethanyG) [inline concepts/tuples/about.md:229]:
```suggestion
[tuple]: https://docs.python.org/3/library/stdtypes.html#tuple
```

File: concepts/tuples/about.md
Comment (BethanyG) [inline concepts/tuples/about.md:228]:
```suggestion
[set]: https://docs.python.org/3/library/stdtypes.html#set
```

**Review by BethanyG — APPROVED** [review]
Alright.  Looks good! 🚀

**Pull request merged**
